packages/clay-css/src/images/icons/content-shield.svg

```diff
@@ -0,0 +1,11 @@
+<!--
+* SPDX-FileCopyrightText: © 2024 Liferay, Inc. <https://liferay.com>
+* SPDX-FileCopyrightText: © 2024 Contributors to the project Clay <https://github.com/liferay/clay/graphs/contributors>
+*
+* SPDX-License-Identifier: BSD-3-Clause
+-->
+<svg viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Right now `viewBox="0 0 16 16"` won't work with the `clay-icon` tag in DXP. It's hardcoded to be `viewBox="0 0 512 512"`. We need the `viewBox="0 0 512 512"` version until we can convert everything to `0 0 16 16`.